default.json

```diff
@@ -16,7 +16,7 @@
         },
         {
             "matchDatasources": ["maven"],
-            "registryUrls": ["https://repox.jfrog.io/repox/sonarsource"]
+            "registryUrls": ["https://repox.jfrog.io/artifactory/sonarsource-releases", "https://repox.jfrog.io/artifactory/central"]
```

Comment:
Why `central` which is included in `sonarsource-releases`?

Reply:
renovate needs to fetch internal dependencies from `sonarsource-releases` and external dependencies from `central` which is a remote repo